src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+
+
+###  http://sweetontology.net/realmGeol/AntiformalStack
+soreag:AntiformalStack rdf:type owl:Class ;
+                       rdfs:subClassOf soreag:StructuralFeature ;
+                       dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                       dcterms:source <http://www.glossaryofgeology.org> ;
+                       skos:definition "A duplex in which the trailing branch lines coincide and the leading branch lines either nearly coincide or lie vertically above each other. This results in a stack of horses and leads to a local culmination in thrust sheet. (Neuendorf et al. (2011) citing Boyer & Elliott (1982))"@en ;
+                       skos:prefLabel "antiformal stack"@en .
+
+
+###  http://sweetontology.net/realmGeol/Arch
+soreag:Arch rdf:type owl:Class ;
+            rdfs:subClassOf soreag:StructuralFeature ;
+            dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            skos:definition "A broad, open anticlinal fold on a regional scale; it is usually a basement doming. (Neuendorf et al., 2011)"@en ;
+            skos:prefLabel "arch"@en .
+
+
+###  http://sweetontology.net/realmGeol/AssemblageZone
+soreag:AssemblageZone rdf:type owl:Class ;
+                      rdfs:subClassOf soreag:BiostratigraphicUnit ;
+                      dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                      dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+                      skos:definition "The body of strata characterized by an assemblage of three or more fossil taxa that, taken together, distinguishes it in biostratigraphic character from adjacent strata. (ICS, Online, December 2019)"@en ;
+                      skos:prefLabel "assemblage zone"@en .
+
+
+###  http://sweetontology.net/realmGeol/AssociationZone
+soreag:AssociationZone rdf:type owl:Class ;
+                       rdfs:subClassOf soreag:BiostratigraphicUnit ;
+                       dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                       dcterms:source <https://doi.org/10.1016/j.revpalbo.2014.04.002> ,
+                                      <https://doi.org/10.1080/01916122.2012.718609> ;
+                       skos:definition "The body of strata between two designated biostratigraphical horizons, which represent the boundaries of the unit and separate it from any preceding or succeeding biozone. These bounding horizons may be delimited individually by: the level of the first stratigraphical appearance (lowermost or oldest occurrence) of a specified taxon (species, genus) in any one stratigraphical section; or another specified biostratigraphical feature considered to approximate a time horizon. The zone is further delimited by specified taxa (preferably, at least three), which have first appearances within the zone. (de Jersey & McKellar, 2013; Bomfleur et al., 2014)"@en ;
```

Comment:
@nicholascar What are your suggestions here please? Do we add ‘biostratigraphic horizon’ (= biohorizon) as a class?

As Stephen has indicated, it is a stratigraphic surface (stratigraphic surface being preferable to his usage of geologic surface) that separates two biozones. But, the way in which ‘biostratigraphic horizon’ is defined in the ICS Code, it is a surface or interface across which there is significant change in biostratigraphic character. As such, it may not necessarily represent a boundary between biozones [e.g., in the hypothetical instance where you have strata with Proterozoic stromatolites being overlain by strata with Devonian plants (where no biozones have been defined)]. But, then, maybe, you wouldn’t necessarily refer to it as a biostratigraphic horizon, although it would represent one [i.e., apart from being an unconformity surface (hiatus)].

For the ontology, the issues are that, apart from biostratigraphic horizons [which we could just refer simply to as boundaries, and perhaps(?) overcome the issue here in this way], there are also lithostratigraphic horizons (lithohorizons), chronostratigraphic horizons (chronohorizons), and magnetostratigraphic polarity-reversal horizons that are all referred to in the Code. Do we bring all of them together under one horizon class (if, at all, we have a class or even classes for these horizons)?

In ontologies for the various biostratigraphic zonations/schemes (yet to be developed by us), biohorizons would represent the horizons/boundaries between the various biostratigraphic units/biozones. In our borehole database, a sample may be assigned to a biostratigraphic zone, and it is always nice to be able to pin point where a biohorizon lies in a stratigraphic succession, but this is generally not achievable to a high degree of accuracy. 

In the case of ‘lithostratigraphic horizon’ (=lithohorizon), this, according to the Code, can be a surface of lithostratigraphic change, and it is commonly the boundary of a lithostratigraphic unit (e.g., the top/bottom of a formation), or a lithologically-distinctive, very-thin marker bed within a lithostratigraphic unit (e.g., in a formation, etc.). Such horizons/boundaries would correspond to the tops and bottoms of formations and other lithostratigraphic units, although, in our borehole database, we do not use the term ‘lithostratigraphic horizon’ (or lithohorizon). We just delimit formation tops (and the tops of other lithostratigraphic units, e.g., group, subgroup, member, etc.) to mark the boundaries between formations, etc. 

As for chronostratigraphic horizon/chronohorizon (a stratigraphic surface or interface that is everywhere of the same age), this term, to my knowledge, doesn’t have much application, except for its relevance to chronostratigraphic units in the Geologic Time Scale, where such horizons define the boundaries between the basic units, i.e., the Stages (mainly for the Phanerozoic where Stages are defined, almost negligibly so for older rocks). But, even here, they are not referred to as chronostratigraphic horizons, although this is what they are.

Finally, we have, in the ICS Code, magnetostratigraphic polarity-reversal horizons: They are surfaces or thin transition intervals (the latter being more than just an horizon) across which the magnetic polarity reverses. [Where the polarity change takes place through a substantial interval of strata, of the order of 1 m in thickness, usage of the term “magnetostratigraphic polarity transition-zone” is advised. Magnetostratigraphic polarity-reversal horizons and polarity-transition zones provide the boundaries for magnetostratigraphic polarity units.

So, to reiterate, do we define all of the above horizons, and if so, is this to be done under one class, with the various subclasses for each type of horizon?